ui.apps/src/main/content/jcr_root/apps/core/cif/components/commerce/productcollection/v2/productcollection/README.md

```diff
@@ -0,0 +1,93 @@
+<!--
+Copyright 2021 Adobe Systems Incorporated
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+Product Collection (v2)
+====
+Product Collection component is a server side component written in HTL, 
+allowing to display a collection of products. 
+
+## Features
+
+* Support for pagination
+* Configurable number of products on one page
+* Client-side loaded prices using GraphQL. Can be disabled in the design dialog.
+* Style System support.
+
+### Use Object
+The Product Collection component uses the `com.adobe.cq.commerce.core.components.models.productcollection.ProductCollection` Sling model as its Use-object.
+
+### Component Policy Configuration Properties
+The following configuration properties are used:
+
+1. `./loadClientPrice` - enables client-side price fetching
+2. `./paginationType` - the pagination type, either `paginationbar` or `loadmorebutton`.
+
+### Edit Dialog Properties
+
+The following properties are written to JCR for this component and are expected to be available as `Resource` properties:
+
+1. `./pageSize` - the number of products shown on one page
+
+## BEM Description
+```
+BLOCK productcollection
+    ELEMENT productcollection__root
+    ELEMENT productcollection__results-count    
+    ELEMENT productcollection__filters
+    ELEMENT productcollection__filters-header
+    ELEMENT productcollection__filters-title
+    ELEMENT productcollection__filters-body
+    ELEMENT productcollection__current-filters
+    ELEMENT productcollection__current-filter
+    ELEMENT productcollection__current-filter-icon
+    ELEMENT productcollection__filter    
+    ELEMENT productcollection__filter-toggler
+    ELEMENT productcollection__filter-header
+    ELEMENT productcollection__filter-icon
+        MOD productcollection__filter-icon--open
+        MOD productcollection__filter-icon--closed    
+    ELEMENT productcollection__filter-items
+    ELEMENT productcollection__filter-item
+    ELEMENT productcollection__filter-title
+    ELEMENT productcollection__items
+    ELEMENT productcollection__item
+        MOD productcollection__item--staged
+    ELEMENT productcollection__item-images
+    ELEMENT productcollection__item-image
+    ELEMENT productcollection__item-image-placeholder
```

Comment:
Isn't that a `MOD` ? At least is see it used in combination with `productcollection__item-image`

Reply:
Indeed, it is. I've fixed it.